include/xbps.h.in

```diff
@@ -540,6 +540,7 @@ struct xbps_handle {
 	 */
 	xbps_array_t preserved_files;
 	xbps_array_t ignored_pkgs;
+	xbps_array_t noextract;
```

Comment:
This will make stuff linked to old libxbps fail (because sizeof(*xhp) won't match).

Anyway as there are no pkgs linking against libxbps, it's not that important.